src/web/components/Avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import { pillarPalette } from '@frontend/lib/pillars';
+
+const contributorImage = css`
+    border-radius: 100%;
+    object-fit: cover;
+    height: 100%;
+    width: 100%;
+`;
+
+const pillarBackground: (pillar: Pillar) => string = pillar => css`
+    background-color: ${pillarPalette[pillar ? pillar : 'opinion'].main};
```

Comment:
We could maybe make `pillar` an optional prop and then default it to 'opinion' instead of using this ternary?